src/components/CardList/CardList.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import Item from './Item';
+
+export interface INodeItem {
```

Comment:
interface properties on lines 4 - 13 should be separated by a semi-colon instead of a comma.
Ex:
`id: number;` not `id: number,`